### PR TITLE
DOC FIX: Move unreleased changelog entries to the correct section

### PR DESCRIPTION
**Pull request by sfc-gh-jkew**
Move plot changelog to pandas section
Move entries from the 1.19.0 pandas section to the 1.19.1 section which were not added to the release notes https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0
Move snowpark-python and local testing entries unpublished with 1.19.0 into the new section as well.

### CODE REVIEW

**Review by sfc-gh-helmeleegy — APPROVED** [review]

**Review by sfc-gh-nkrishna — COMMENTED** [review]
Would it be possible to do the same for Snowpark Python + local testing (the unreleased features) -- I think theirs is affected as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -80,42 +102,22 @@
 - Added support for `DataFrame.expanding` and `Series.expanding` for aggregations `count`, `sum`, `min`, `max`, `mean`, `std`, `var`, and `sem` with `axis=0`.
 - Added support for `DataFrame.rolling` and `Series.rolling` for aggregation `count` with `axis=0`.
 - Added support for `DataFrameGroupBy.get_group`.
```
Comment (sfc-gh-yzou) [inline CHANGELOG.md:104]: the get_group seems also not in the 1.19.0 release according to the diff

File: CHANGELOG.md [hunk 2]
```diff
@@ -80,42 +102,22 @@
...
-- Added support for `DataFrameGroupBy` and `SeriesGroupBy` aggregations `first` and `last`
 - Added support for `Series.str.match`.
 - Added support for `DataFrame.resample` and `Series.resample` for aggregations `size`, `first`, and `last`.
-- Added support for `DataFrameGroupBy.all`, `SeriesGroupBy.all`, `DataFrameGroupBy.any`, and `SeriesGroupBy.any`.
-- Added support for `DataFrame.nlargest`, `DataFrame.nsmallest`, `Series.nlargest` and `Series.nsmallest`.
-- Added support for `replace` and `frac > 1` in `DataFrame.sample` and `Series.sample`.
```
Comment (sfc-gh-yzou) [inline CHANGELOG.md:88]: based on our release cut change log, https://github.com/snowflakedb/snowpark-python/blob/release-v1.19.0/CHANGELOG.md most of those changes seems already in on the release branch

File: CHANGELOG.md [hunk 3]
```diff
@@ -80,42 +102,22 @@
...
 
 #### Improvements
 
 - Added support for named aggregations in `DataFrame.aggregate` and `Series.aggregate` with `axis=0`.
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an `pd.Index` directly in Snowpark pandas. Support for `pd.Index` as a first-class component of Snowpark pandas is coming soon.
 - Added a lazy index constructor and support for `len`, `shape`, `size`, `empty`, `to_pandas()` and `names`. For `df.index`, Snowpark pandas creates a lazy index object.
-- For `df.index`, Snowpark pandas creates a lazy index object. 
 - For `df.columns`, Snowpark pandas supports a non-lazy version of an `Index` since the data is already stored locally.
-- Added support for `Index.copy()`
```
Comment (sfc-gh-yzou) [inline CHANGELOG.md:115]: based on the change log on the release cut, those change seems already in

File: CHANGELOG.md (hunk 2 above)
Comment (sfc-gh-jkew) [reply CHANGELOG.md:88]: Oh. I see. Our release notes are also not in sync here: https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0

Comment (sfc-gh-jkew) [reply CHANGELOG.md]: Ok. I based the most recent changes off of the changelog in the branch as opposed to the release notes.

File: CHANGELOG.md [hunk 4]
```diff
@@ -93,8 +104,6 @@
 - Added documentation pages for `Index` and its APIs.
 - Added support for `DataFrame.assign`.
 - Added support for `DataFrame.stack`.
-- Added support for `DataFrame.pivot` and `pd.pivot`.
```
Comment (sfc-gh-yzou) [inline CHANGELOG.md:96]: Oh, @sfc-gh-jkew sorry, actually, snowpark python seems use the release tag commit hash here https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0, what you had before seems correct.

**Comment by sfc-gh-yzou** [comment]
@sfc-gh-nkrishna yes, let me work with @sfc-gh-aalam to go over the release log one more time to do the correction for snowpark python

File: CHANGELOG.md (hunk 4 above)
Comment (sfc-gh-jkew) [reply CHANGELOG.md:96]: hmm. We may need to ask for help. Originally I was diffing with the release tag; but I think diffing with the branch still makes sense like you said in your first comment.

**Comment by sfc-gh-jkew** [comment]
@sfc-gh-aalam @sfc-gh-yzou This should reflect the diff with the release notes tag as opposed to the changelog in the branch.

File: CHANGELOG.md [hunk 5]
```diff
@@ -80,42 +114,22 @@
 - Added support for `DataFrame.expanding` and `Series.expanding` for aggregations `count`, `sum`, `min`, `max`, `mean`, `std`, `var`, and `sem` with `axis=0`.
 - Added support for `DataFrame.rolling` and `Series.rolling` for aggregation `count` with `axis=0`.
 - Added support for `DataFrameGroupBy.get_group`.
```
Comment (sfc-gh-yzou) [inline CHANGELOG.md:116]: I don't see get_group here https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0

**Review by sfc-gh-yzou — APPROVED** [review]
i did one pass compare with the release log here https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0, there seems one mismatch for get_group

**Pull request merged**